internal/lexer/lexer_test.go

```diff
@@ -0,0 +1,83 @@
+package lexer_test
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+	"text/scanner"
+
+	"github.com/neelance/graphql-go/internal/lexer"
+)
+
+func TestLexer(t *testing.T) {
```

Comment:
1: completely agree, but wanted to play a little bit around
2: `array of floats` was failing before